gigadb/app/tools/excel-spreadsheet-uploader/execute.sh

```diff
@@ -3,12 +3,28 @@
 PATH=/usr/local/bin:$PATH
 export PATH
 
-docker-compose run --rm pg_client -c 'drop trigger if exists file_finder_trigger on file RESTRICT'
-docker-compose run --rm pg_client -c 'drop trigger if exists sample_finder_trigger on sample RESTRICT'
-docker-compose run --rm pg_client -c 'drop trigger if exists dataset_finder_trigger on dataset RESTRICT'
+if [[ $(uname -n) =~ compute ]];then
+  . /home/centos/.bash_profile
+  docker run --rm  --env-file ./db-env registry.gitlab.com/gigascience/forks/rija-gigadb-website/production_pgclient:$GIGADB_ENV -c 'drop trigger if exists file_finder_trigger on file RESTRICT'
+  docker run --rm  --env-file ./db-env registry.gitlab.com/gigascience/forks/rija-gigadb-website/production_pgclient:$GIGADB_ENV -c 'drop trigger if exists sample_finder_trigger on sample RESTRICT'
+  docker run --rm  --env-file ./db-env registry.gitlab.com/gigascience/forks/rija-gigadb-website/production_pgclient:$GIGADB_ENV -c 'drop trigger if exists dataset_finder_trigger on dataset RESTRICT'
+
+  docker run --rm -v /home/centos/uploadDir:/tool/uploadDir -v /home/centos/uploadLogs:/tool/logs registry.gitlab.com/gigascience/forks/rija-gigadb-website/production_xls_uploader:$GIGADB_ENV ./run.sh
```

Comment:
@rija your GigaDB Gitlab project name `rija-gigadb-website` shouldn't be hardcoded here?

Maybe this is why I can't get spreadsheet upload working on my staging server?
```
line:     data text
yiisession
org.postgresql.util.PSQLException: The connection attempt failed.
        at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:152)
        at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:66)
        at org.postgresql.jdbc2.AbstractJdbc2Connection.<init>(AbstractJdbc2Connection.java:125)
...
Caused by: java.net.UnknownHostException: rds-server-staging-rija.cwbzwlupzz1e.eu-west-3.rds.amazonaws.com
        at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:184)
...
```

Reply:
Hi @pli888 

Thanks for spotting that issue. I have pushed a fix for it.
The bastion playbook will now add an extra variable (GITLAB_PROJECT) in the ``/home/centos/.bash_profile``, and the ``execute.sh`` script will make use of it instead of the hardcoded string in container image name.
That  required a change to ``ops/infrastructure/inventories/hosts`` to allow the gitlab project name to be available for interpolation in the Ansible playbook.